src/interface/storage.thrift

```diff
@@ -495,6 +495,7 @@ struct LookupIndexResp {
     // Each column represents one property. the column name is in the form of "tag_name.prop_alias"
     // or "edge_type_name.prop_alias" in the same order which specified in return_columns of request
     2: optional common.DataSet          data,
+    3: optional common.DataSet          stat_data,
```

Comment:
Base on implementation, stat_data only has one row, each column in the row is one stat, right?

Maybe add some brief description in thrift file